### PR TITLE
Named jobs within single build stage

### PR DESCRIPTION
**Pull request by harbu**
Seems to cut off few minutes from build.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,12 +10,10 @@ before_install:
...
-    - npm run test-unit
-  - stage: Integration tests
-    script:
-    - npm run test-integration
\ No newline at end of file
+  - stage: tests
+    name: lint
+    script: npm run eslint
+  - script: npm run test-unit
+    name: unit tests
```
Comment (mirpo) [inline .travis.yml]: is it correct - first script then name?

Comment (harbu) [reply .travis.yml]: yes

**Comment by harbu** [comment]
Trying changing order. Let's see if Travis CI likes this.

**Comment by harbu** [comment]
https://docs.travis-ci.com/user/build-stages#naming-your-jobs-within-build-stages

**Comment by harbu** [comment]
Tests pass, @mirpo , okay to merge?

**Pull request merged**